src/nl/hannahsten/texifyidea/util/SystemEnvironment.kt

```diff
@@ -110,3 +110,20 @@ fun runCommandWithExitCode(vararg commands: String, workingDirectory: File? = nu
         }
     }
 }
+
+fun runCommandWithoutReturn(vararg commands: String, workingDirectory: File? = null) {
```

Comment:
It is not clear to me why you need this method, it looks like a bad idea to me as you're just duplicating the code but skipping the timeout functionality?

Reply:
My problem was that the inputstream or the errorstream is blocking the execution with certain commands. I added a function to not read the streams. But, it might not have been the best way of doing it. I commited a new version of runCommandWithExitCode that offers non-blocking execution. So that when the inputstream or the errorstream isn't ready to be read, it doesn't read them. I use it in ConfigureInverseSearchAction and in SumatraConversation in openFile, places where I don't need the output in the streams. Let me know what you think of this change.

Reply:
That is a lot clearer to me, thanks!